spec/models/govspeak_decorator_spec.rb

```diff
@@ -20,12 +20,13 @@
     expect(rendered).to eq(expected_html)
   end
 
-  it 'returns kramdown doc if sanitize false' do
-    govspeak_md = '$YOUTUBE(EpjSlCJtPLos)$YOUTUBE'
-    expected_html = %(<h2 class="govuk-heading-m govuk-!-margin-top-2"></h2>\n<div class="govspeak-embed-container" style="padding:56.19% 0 0 0;position:relative;"><iframe class="govspeak-embed-video" style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/EpjSlCJtPLos?enablejsapi=1&amp;origin=#{ENV['DOMAIN']}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>\n)
-    rendered = described_class.translate_markdown(govspeak_md, sanitize: false)
-    expect(rendered).to eq(expected_html)
-  end
+  #  Not sure about this spec
```

Comment:
Stringifying it, which happens in the view automatically, with `#to_s` should get this working. I hope.

Reply:
It's not that it doesn't work, it passes, it just looked like it wasn't doing anything - didn't understand the kramdown doc thing because on inspect it looked it was returning the same html as other similar tests